src/x_connective.c

```diff
@@ -156,6 +156,13 @@ static void pdsymbol_symbol(t_pdsymbol *x, t_symbol *s)
     outlet_symbol(x->x_obj.ob_outlet, x->x_s = s);
 }
 
+static void pdsymbol_float(t_pdsymbol *x, t_float f)
+{
+    char s[14];
+    sprintf(s, "%g", f);
+    outlet_symbol(x->x_obj.ob_outlet, gensym(s));
+}
```

Comment:
this fails to implement the core functionality of `[symbol]` - that is: to store the actual symbol.
this kind of "breaks" the object.

Reply:
oops... yeah, this just seems like a timid starting point for implementing a proper way to deal with floats, far from done. You also had remarks on this code as well, huh? 

I can go ahead and try to make this a proper PR myself if I get some help on what to do.

cheers

Reply:
i thought this was already meant as a "proper" PR. if you just want a branch for discussion, please add `[WIP]` at the beginning of the title.

Reply:
what does [WIP] mean?